frontend/public/components/namespace.jsx

```diff
@@ -1044,106 +1110,104 @@ const namespaceBarDropdownDispatchToProps = (dispatch) => ({
   showStartGuide: (show) => dispatch(setFlag(FLAGS.SHOW_OPENSHIFT_START_GUIDE, show)),
 });
 
-class NamespaceBarDropdowns_ extends React.Component {
-  componentDidUpdate() {
-    const { namespace, showStartGuide } = this.props;
+const NamespaceBarDropdowns_ = (props) => {
+  const {
+    activeNamespace,
+    canCreateProject,
+    canListNS,
+    children,
+    disabled,
+    namespace,
+    onNamespaceChange,
+    setActiveNamespace,
+    showStartGuide,
+    t,
```

Comment:
Since this is a function component now, you can leverage the `useTranslation` hook and get rid of `withTranslation` HOC

Reply:
Removed in next commit